draft-ietf-tls-tls13.md

```diff
@@ -5636,6 +5759,72 @@ application protocols separately ensuring that confidential
 information is not inadvertently leaked.
 
 
+## Replay Attacks on 0-RTT {#replay-0rtt}
+
+Replayable 0-RTT data presents a number of security threats to
+TLS-using applications. Specifically, if applications are not
+engineered to be replay safe (minimally, this means idempotent, but in
+some cases may there might be even stronger conditions), then duplication of
+actions will cause side effects (e.g., purchasing an item or
+transferring money) to be duplicated, thus harming the site or the
+user.  Attackers can also store and replay 0-RTT messages in order to
+re-order them with respect to other messages, which may harm
+unsuspecting applications.
+
+If data can be replayed a large number of times, additional attacks
+become possible. Specifically, attackers can use multiple replays to
+exploit information leakage via side channels such as timing network
+caches or measuring the speed of cryptographic operations. In addition, they may
```

Comment:
(Just noting on the PR that Ilari sent mail noting that some of these should move to the previous paragraph)